app/config/config.yml

```diff
@@ -129,3 +129,7 @@ mongo_db_migrations:
      script_dir_name: "%kernel.root_dir%/scripts"
      name: "Open Orchestra MongoDB Migrations"
      namespace: OpenOrchestra\ModelBundle\Migrations\MongoDB
+
+bazinga_js_translation:
+    active_domains:
+        - messages
```

Comment:
@alavieille : what do you think about introducing a dedicated domain to the Js client to avoid confusion between server translations and client translations ?